tools/dataProtector/dataProtectorCore/revokeAllAccess.md

```diff
@@ -49,12 +49,12 @@ const revokeAllAccessResult = await dataProtectorCore.revokeAllAccess({
 
 ### authorizedApp
 
-`AddressOrENS | 'any' | undefined`
-
-_default_: `any`
+`AddressOrENS | undefined`
 
 The application address to be removed from the authorization list for the
-specified `protectedData`.
+specified `protectedData`. If no address is specified, it will return the
+granted access that authorized any app address (equivalent to
+`0x0000000000000000000000000000000000000000`).
```

Comment:
```suggestion
specified `protectedData`. If no address is specified, it will revoke all access
from the protected data, regardless of the app.
```

Open question:
If I have given access to my protected data to 3 apps:
 - "0x123..."
 - "0x456..."
 - "0x000..."
(The 2 first access are not really useful since the 3rd one is on the Zero address.)

If I choose to revokeAllAccess with authorizedApp: "0x000...", I believe the two first access remain, right?